src/Core/Application/Dependency/Repository/TaskSetTaskRelationRepositoryInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Dependency\Repository;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\TaskSetTaskRelationInterface;
+
+interface TaskSetTaskRelationRepositoryInterface
```

Comment:
Do we need this repo for one-to-many? Can we resolve it with annotations in task model?